package.json

```diff
@@ -112,7 +119,7 @@
     "uglifyjs-webpack-plugin": "1.1.4",
     "url-loader": "0.5.8",
     "uuid": "3.0.1",
-    "webpack": "2.7.0",
+    "webpack": "3.11.0",
```

Comment:
bumping to Webpack 3 allows to take advantage of its new scope-hoisting feature, which promises to speed up our bundle's initial execution time. (this is Rollup's claim to fame.)